datalad_debian/configure_builder.py

```diff
@@ -25,12 +25,38 @@
 
 @build_doc
 class ConfigureBuilder(Interface):
-    """Create a new package dataset
+    """Configure a package build environment
+
+    A builder is a (containerized) build environment used to build binary Debian
+    packages from source files. This command is typically run in a distribution
+    dataset and configures a builder recipe based on a template and
+    user-specified values for the template's placeholders.
+    The resulting recipe will be placed in the 'recipes/' directory in the
+    'builder/' subdataset of a distribution dataset.
+
+    The following directory tree illustrates this with an example. The
+    configured builder takes the form of a Singularity recipe here.
+
+        | bullseye                <- distribution dataset
+        |    ├── builder             <- builder subdataset
+        |    │   ├── envs
+        |    │   │   └──  README.md
+        |    │   └── recipes
+        |    │       ├── README.md
+        |    │       └── singularity-any     <- configured builder recipe
+
+    Currently supported templates are
+
+    - 'singularity-default': A Singularity recipe requiring the spec
+      'dockerbase' with a value for containers' base image
+
     """
+
     _params_ = dict(
         dataset=Parameter(
             args=("-d", "--dataset"),
-            doc="""specify a distribution dataset to add the package to""",
+            doc="""Specify a builder dataset in which an environment will be
+            configured""",
```

Comment:
I'm bumping on the word "configured" here. Perhaps "an environment configuration will be defined"? I'm unsure...

Reply:
Used your suggestion